templates/index.html

```diff
@@ -3,55 +3,165 @@
+            of the way there
+            <em>(as of {{ stats.update_time.strftime('%B %-d at %-I:%M %p EST') }})</em>.
+          </p>
       </div>
     </div>
-    <div class="form-row">
-      <div class="col-md-12 col-sm-12 mb-3">
-        Optional fields:
-      </div>
+
+    <div>
+      <h3 class='text-center'>Do your part: Help your friends in Georgia vote</h3>
+      <hr class='my-5'>
     </div>
-    <div class="form-row">
-      <div class="col-md-4 col-sm-12 mb-3">
-        <label for="input-first">Middle name/initial</label>
-        <input type="text" class="form-control" placeholder="Middle name" name="middle" id="input-middle">
+
+    <div>
+      <!-- TODO: Have the # be its own column and everything else to the right for a bit more separation? Depends if we are cool with the wider container though. -->
+      <h4>1️⃣ Who do you know in Georgia?</h4>
+      <p>Check which of your Facebook friends lives in Georgia.</p>
+      <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia"
+        target="_blank" rel="noopener" class="btn text-white mb-3"
+        style="background-color: #e12814;">Find friends in Georgia on Facebook</a>
+      <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click
+          the button above to run the search, or just go to Facebook and type "my friends in
+          Georgia" into the search box.</em></p>
+      <hr class="my-5">
+    </div>
+
+    <div class="mt-4">
+      <h4>2️⃣ Let them know there's an election</h4>
+      <p>In most elections, over 30% of voters don't know when the election is. Make sure your
+        friends know that the election is on <strong>Tuesday, January 5, 2021</strong>. Anyone can
+        vote early in-person starting on <strong>Monday, December 14, 2020</strong>.</p>
+
+      <div class="d-flex flex-column justify-content-end align-items-end">
+        <div class="card bg-primary rounded-lg text-white mb-2" style="width: 18rem;">
+          <div class="card-body">
+            <p class="card-text">
+              Hi Emily! I'm sure you already know that the Georgia runoff election, the one that
+              determines who controls the Senate, is on January 5th. I just wanted to pass along
+              this link to find your early voting location -- you can start voting today!<br><br>
+              <strong>www.voteamerica.com/where-to-vote</strong></p>
+          </div>
+        </div>
+
+        <!-- TODO: See if this works, I have an Android rn -->
+        <a href="sms://body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F"
```

Comment:
on a mac (running Big Sur), it successfully opens the messages app. it does NOT successfully prepopulate the text of the message. i believe you need an &amp; before the word "body" - this works for me. (via https://stackoverflow.com/questions/47408196/pre-populate-the-sms-body-text-with-an-https-url)

```suggestion
        <a href="sms://&body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F"
```

Reply:
thx!